packages/wdio-cli/src/templates/wdio.conf.tpl.ejs

```diff
@@ -232,7 +251,13 @@ if (answers.isUsingTypeScript) {
     // specFileRetriesDeferred: false,
     //
     <%- include('snippets/reporters', { reporters: answers.reporters }) %>
-    <% if(answers.framework === 'mocha') { %>
+    <% if(answers.serenityAdapter === 'mocha') { %>//
```

Comment:
Can we move all the framework opts into their own template snippets?

Reply:
Yeah, good call; let me do this now.

Reply:
Done in https://github.com/webdriverio/webdriverio/pull/11368/commits/68a437c2e7894a647e9ff03e277fc8ad046dcf0e#diff-cc58b213dd871b45469e751b91589bd4836c606b4dbb8af89fc9cc38782fec66